javascript/boilerplate-text/boilerplate-fr.js

```diff
@@ -115,6 +115,15 @@ s.validCSS = "CSS valide !";
 s.codedInUtf8 = "Codé en UTF-8 !";
 
 s.translatedFromEnglishVer = "Traduit d’un contenu en anglais daté du "+dt.enVersion+". Dernière modification de cette traduction le "+dt.thisVersionPlain+" UTC."; 
+// this block of strings should include any whitespace needed after the colon
+// or its equivalent, eg. "By: " in english, but "作者：" in chinese
+s.author = "Par : "; // followed by name of author(s)
+s.previousAuthors = "Précédemment par : "; // followed by name of previous author(s), before substantive changes were made by the current author
+s.modifiedBy = "Modifié par : "; // person's name appears after colon
+s.translatedBy = "Traducteur : ";
```

Comment:
Fwiw, this came up recently during the Galician translation. Since the translator was female, she wondered whether the translation should be Traductor/traductora (here i guess that equates to Traducteur/traductrice).  We finally settled on Traducido por (i guess in French, Traduit par) to keep things simple.  Should that apply here ?

Reply:
Indeed. There is a new gender-neutral term in French "Traductaire", let me think about it.